src/main/java/graphql/testrunner/service/GitService.java

```diff
@@ -22,13 +23,13 @@ public class GitService {
     @Autowired
     private Git git;
 
-    public void checkout(String commitHash) throws TestRunnerException {
+    public void checkout(Job job) throws TestRunnerException {
```

Comment:
there is context in `Job` that this method doesn't need. let's explicitly pull out the context it does, and pass them in as arguments.

Reply:
Done.